networking/network_policy/about-network-policy.adoc

```diff
@@ -1,7 +1,12 @@
 :_content-type: ASSEMBLY
 [id="about-network-policy"]
 = About network policy
+ifdef::openshift-enterprise[]
 include::_attributes/common-attributes.adoc[]
```

Comment:
Is there a specific reason why this attribute file should only be included in the `openshift-enterprise` distro? The attribute file might be required by other distros, such as OKD. Also, my understanding is that for OSD and ROSA, the attributes in the subsequent `_attributes/attributes-openshift-dedicated.adoc` file would be used instead of the attributes in `_attributes/common-attributes.adoc`, given that `_attributes/attributes-openshift-dedicated.adoc` it is specified last.